InteractiveHtmlBom/web/ibom.js

```diff
@@ -659,7 +659,17 @@ function populateBomBody(placeholderColumn = null, placeHolderElements = null) {
           var valueSet = new Set();
           references.map(r => r[1]).forEach((id) => valueSet.add(pcbdata.bom.fields[id][field_index]));
           td = document.createElement("TD");
-          td.innerHTML = highlightFilter(Array.from(valueSet).join(", "));
+          var output = new Array();
+          const urlRegex = /^((file|https?):\/\/(www\.)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b([-a-zA-Z0-9()@:%_\+.~#?&//=]*))$/g;
```

Comment:
Move this constant to top of the class.

Regex for file:// needs to be relaxed, there are no length limits or dot requirements in file paths, also more characters are allowed like spaces. In general make it separate from http(s). Something like `(https?:....|file:....)`. 

This regex is good enough https://gitlab.com/kicad/code/kicad/-/blob/2f8cc845513b918e219f32733a210bf458138f91/kicad/pcm/schemas/pcm.v1.schema.json#L287

Reply:
> Move this constant to top of the class.

Can you please point me to the class you are referring to. I could not find a JS class.
I just moved the definition to the top of the function. Please get back to me if you prefer a different location.

> This regex is good enough https://gitlab.com/kicad/code/kicad/-/blob/2f8cc845513b918e219f32733a210bf458138f91/kicad/pcm/schemas/pcm.v1.schema.json#L287

Looks good at first glance, but I see potential issues. Would you mind reviewing my commit message and maybe consider improving the regex in the original code? See 9c1f97ceed9c3cf070f68ea56e34953dc819a5e4